isis/src/base/objs/History/unitTest.cpp

```diff
@@ -17,14 +17,14 @@ using namespace std;
 void IsisMain() {
   Isis::Preference::Preferences(true);
 
-  Isis::History h("Haha");;
-  h.AddEntry();
-  QString file = "unitTest.tttt";
-  h.Write(file);
-
-  Isis::History h2("Haha", file);
-  Isis::PvlObject o = h2.ReturnHist();
-  std::cout << o << std::endl;
-
-  remove(file.toLatin1().data());
+  // Isis::History h("Haha");;
+  // h.AddEntry();
+  // QString file = "unitTest.tttt";
+  // h.Write(file);
+  //
+  // Isis::History h2("Haha", file);
+  // Isis::PvlObject o = h2.ReturnHist();
+  // std::cout << o << std::endl;
+  //
+  // remove(file.toLatin1().data());
```

Comment:
I'm working on writing new tests for the history object along with the functions I've added.